.github/workflows/openms-ci.yml

```diff
@@ -94,8 +102,10 @@ jobs:
           WITH_GUI: "ON"
           ADDRESS_SANITIZER: "Off"
           BUILD_TYPE: "Release"
-          OPENMP: "On"
+          OPENMP: "Off"
```

Comment:
Not sure either - I thought its because of this. @jpfeuffer why did you disable it?

Reply:
I hoped that it would reduce compile times